server/interceptor/interceptor.go

```diff
@@ -39,10 +45,12 @@ type TrillianInterceptor struct {
 }
 
 // UnaryInterceptor executes the TrillianInterceptor logic for unary RPCs.
-func (i *TrillianInterceptor) UnaryInterceptor(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
-	quotaUser := i.QuotaManager.GetUser(ctx, req)
+func (i *TrillianInterceptor) UnaryInterceptor(ctx context.Context, req interface{}, _ *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+	// IMPORTANT: Do not rely on grpc.UnaryServerInfo in this filter. It makes life a lot harder
+	// when adapting the code to our internal branch.
```

Comment:
Maybe "when adapting the code to other environments."  (ditto for commit message), to avoid an us/them distinction.